exercises/practice/gigasecond/.docs/instructions.append.md

```diff
@@ -0,0 +1,8 @@
+# Instructions append
+
+## Odin-specific instructions
+
```

Comment:
The level-1 heading is ignored by the exercism backend code that generates the instructions (or the README.md). 
The level-2 heading remains, and is the correct heading level at that point.